<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaConMenues</class>
 <widget class="QMainWindow" name="VentanaConMenues">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaConMenues</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/recursos/donat.png</normaloff>:/recursos/donat.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="LineaDeTexto" name="lineaGoogle">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="LineaDeTexto" name="lineaLista">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="separator"/>
    <addaction name="actionImprimir"/>
    <addaction name="separator"/>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuEdici_n">
    <property name="title">
     <string>Edición</string>
    </property>
    <addaction name="actionCopiar"/>
    <addaction name="actionPegar"/>
    <addaction name="actionCortar"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionHerramientas"/>
    <addaction name="actionConfiguraci_n"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdici_n"/>
   <addaction name="menuOpciones"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionGuardar"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset>
     <normaloff>:/recursos/nuevo.png</normaloff>:/recursos/nuevo.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset>
     <normaloff>:/recursos/guardar.png</normaloff>:/recursos/guardar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="text">
    <string>Guardar como</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionImprimir">
   <property name="text">
    <string>Imprimir</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="text">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionHerramientas">
   <property name="text">
    <string>Herramientas</string>
   </property>
  </action>
  <action name="actionConfiguraci_n">
   <property name="text">
    <string>Configuración</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineaDeTexto</class>
   <extends>QLineEdit</extends>
   <header>lineadetexto.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
